R/label.R

```diff
@@ -0,0 +1,34 @@
+#' @title Label
+#' @docType class
+#' @description
+#' Label class associate font to a text
+#' @include utils.R
+#' @export
+Label <- R6::R6Class(
+  "Label",
+  public = list(
+    text = NULL,
+    font = NULL,
+
+    initialize = function(text = "", font = NULL) {
+      self$text <- text
+      validateIsOfType(font, Font, nullAllowed = TRUE)

```

Comment:
This is how it's being used. You can pass the type as a string or the type directly
By default, nulI is not allowed but you can specify true when the parameter is optional